pkg/reconciler/configmappropagation/resources/labels.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+const (
+	PropagationLabelKey           = "knative.dev/config-propagation"
+	PropagationLabelValueOriginal = "original"
+	PropagationLabelValueCopy     = "copy"
+)
+
+func OriginalLabels(selector map[string]string) map[string]string {
```

Comment:
This function looks quite misleading to me:
1. it mutates the argument in place but returns a map, giving the false impression that a copy is returned
2. it sets something, but doesn't include "Set" in the name

Is it truly necessary?

Reply:
I'll pass the deepcopy in this method, not the original one (It doesn't need to set something, my mistake.)